mobile/src/utils/hooks/useDeviceVerification.ts

```diff
@@ -75,6 +85,20 @@ export function useDeviceVerification(shouldVerify: boolean) {
       } catch (e) {
         console.log("error in verifyDevice", e);
```

Comment:
Should this not stop here and return an error as verifying did not work. Also if we still want to log this to the user's console, please change it to console.error then

Reply:
Unused file. removed.